src/compiler/checker.ts

```diff
@@ -11072,9 +11120,9 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
         if (parentAccess && canHaveFlowNode(parentAccess) && parentAccess.flowNode) {
             const propName = getDestructuringPropertyName(node);
             if (propName) {
-                const literal = setTextRange(parseNodeFactory.createStringLiteral(propName), node);
+                const literal = setTextRangeWorker(parseNodeFactory.createStringLiteral(propName), node);

```

Comment:
Seems weird to rename this elsewhere; in other files we just import the symbol twice, once with another name like `ts_setTextRange`; maybe we do that here to not have to change anything outside of the node builder?

Reply:
Eh, there were only something like 3 uses outside of the node builder and, if I'm honest, they *probably* need the same sanity checking the node builder impl does, but I didn't want to put in the effort to prove it yet. (Why do the synthetic nodes manufactured for control flow even need a text range set? I honestly have no clue what it's used for. Errors?)

Anyways, the diff from the import rename isn't that large, since it's not used much outside the node builder - we'd just be trading 3 lines of diff for one line added.

Reply:
Alright, that's fine too.